tests/gitlab/build_and_test

```diff
@@ -205,9 +238,14 @@ then
     export SPACK_USER_CACHE_PATH="${spack_user_cache}"
     mkdir -p ${spack_user_cache}
 
-    echo "Fetching uberenv."
+    echo "Fetching uberenv ..."
+    # FIXME: the messages from the next clone appear out-of-place, at the end of
+    #        the logs on Gitlab. (On Lassen the messages are where expected.)
```

Comment:
This is just an odd behavior I noticed on Ruby and Tioga. It looks like an issue with how the batch system merges stdout and stderr.